README.md

```diff
@@ -8,10 +8,43 @@ Poky GENIVI compliant.
 Branch Policy
 -------------
 
-Please note:
-- The most recent development is done on the highest numbered branch
-- Master branch is used as a stable version, typically equal to the most recently *released* numbered branch
-- Maintenance patches are added to both master and released branches
+- New development is done on **master** branch and most new change requests
+(PRs) should be proposed as changes to master, unless you know they are
+applicable to a particular release only.
+- Somewhere near a new major release, a numbered release branch (e.g. 14.0)
+is created from master, and the new branch goes into stabilization/release phase.
+- Release numbers use [semantic versioning](http://semver.org/), and are
+potentially also tagged with GENIVI-specific release tags, such as "M-0.2"
+for the second baseline prerelease for the "M" platform release.
+- After a versioned release, the release branch remains for maintenance and
```

Comment:
Are you trying to describe the current numbering scheme or explain a new one? Doesn't the current numbering scheme not strictly follow semantic versioning, as the major number changes for each cycle regardless of whether it is back incompatible API or not?

As an aside I was never particularly fond of having both the A-x.y and x.y.z numbering scheme for the baseline. I thought it could be confusing for those not familiar with the release process, although I realise the A-x.y releases as to facilitate the two pre-releases.

It can also be a little confusing when you look at the meta-ivi tags and find for example 12.0.0 and N-1.0 are not exactly equivalent due the former having a different distro version number. This might be an opportunity to tighten up the release process, i.e. the distro number should be bumped on N-1.0 so they are equivalent. I may be forgetting some subtlety though. 

Back on the diff, maybe we should just be more specific here and say there will be two A-x.y pre-releases, followed by x.y.z releases at the point of A-1.0. Alternatively it could point at something more specific on-line, but it feels like we ought to be able to do it here.

Reply:
The semantic versioning clarification in the update deals with the concern I had